src/constants.js

```diff
@@ -52,4 +54,10 @@ export const componentKeys = {
   TREATMENTS: "TR",
   NURSING_TASKS: "NT",
   DRUG_CHART: "DC",
-};
\ No newline at end of file
+};
+
+export const items = [
```

Comment:
this is is global constant file, something which would get used at multiple places... as this is specific to the Nursing task... can you please move it to Nursing Task folder?

Reply:
You mean under this path - src/features/DisplayControls/NursingTasks/utils ?